contracts/voting-verifier/src/contract/execute.rs

```diff
@@ -256,19 +275,25 @@ pub fn vote(
 }
 
 pub fn end_poll(deps: DepsMut, env: Env, poll_id: PollId) -> Result<Response, ContractError> {
-    let config = CONFIG.load(deps.storage)?;
+    let config = CONFIG
+        .load(deps.storage)
+        .change_context(ContractError::StorageError)?;
```

Comment:
should we change loading the config to a panic instead of result? Unless there is something disastrously wrong with the contract, the config should always be loadable

Reply:
I am all for that, I dislike treating this as an error